WatchlistMovie+CoreDataProperties.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  WatchlistMovie+CoreDataProperties.swift
+//  Movies
+//
+//  Created by Diana Chizhik on 10/08/2022.
+//
+//
+
+import Foundation
+import CoreData
+
+struct WatchlistMovieConfiguration {
```

Comment:
This may be a bit of personal opinion but I'm not sure about this naming choice, to be a "Configuration" type. I'd rather go to something like `WatchlistMovieModel` or simply `WatchlistMovie` and having the core data entity as `CDWatchlistMovie`.

Reply:
If I'm asked what my weaknesses are, I'll say naming :)
Well, I don't think changing the name of CoreData entity is a good idea. I think it has once already ruined everything.
Let's discuss it in the call, shall me?